plugins/Satellites/ChangeLog

```diff
@@ -1,3 +1,5 @@
+include "no group assigned" secundary filter

```

Comment:
add version/date and fix typo "secondary".

Reply:
### The plugin vs. catalog version string is a source of confusion

Also, there are **two** _catalog_ versions for the format: `0.12.0` and `0.12.4` ; maybe a small oversight?

The _JSON file format version_ and  _plugin_ version are not immediately related. One can change without the other changing, _sometimes_ one _does_ influence the other.

https://github.com/Stellarium/stellarium/blob/31611416a8f02163bf42b05ee80c3b670ef992c7/plugins/Satellites/CMakeLists.txt#L1

https://github.com/Stellarium/stellarium/blob/31611416a8f02163bf42b05ee80c3b670ef992c7/plugins/Satellites/src/Satellites.cpp#L85

https://raw.githubusercontent.com/Stellarium/stellarium/master/plugins/Satellites/resources/satellites.json contains _two_ version strings??

**Proposition:** 
Separate _plugin_ version shown in the plugin dialog (make it `1.1.0`) from the catalog format version ? (`0.12.4` is currently shown in the Satellite dialog, but that is the _catalog format_ version... If needed, include versions of plugin as well as catalog format ?).

The "catalog format version" would only change when the format changes (what is exactly what the value means).

Reply:
I was referring to entries in exactly this file which show some version number and date. The formatting of your ChangeLog entry does not match the other entries. It can be assumed that the version referred to in the ChangeLog is the plugin version.